Microsoft.Quantum.Katas/Utils.qs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+//////////////////////////////////////////////////////////////////////
+// This file contains some Q# intrinsic operations
+// needed for the validation of some Katas' answers.
+//////////////////////////////////////////////////////////////////////
+
+namespace Quantum.Kata.Utils {
+    
+    /// # Summary
+    /// Returns how many times a given oracle is executed.
+    function GetOracleCallsCount<'T> (oracle : 'T) : Int { body intrinsic; }
+    
+    /// # Summary
+    /// Resets the variable that tracks how many times an oracle
+    /// is executed back to 0.
+    function ResetOracleCallsCount () : Unit { body intrinsic; }
```

Comment:
Similary, these all need to be operations as well, as they are not deterministic.